pkg/service/model_build_managed_sg.go

```diff
@@ -113,6 +129,9 @@ func (t *defaultModelBuildTask) buildCIDRsFromSourceRanges(_ context.Context, ip
 		}
 	}
 	if len(cidrs) == 0 {
+		if prefixListsConfigured {
```

Comment:
just for my understanding - why we allow an empty cidr, if prefixLists is configured?

Reply:
If the user just wants to allow the traffic only from the prefix list, the controller needs to skip the unlimited CIDR settings (`0.0.0.0/0` or `::/0`)